### PR TITLE
Download checkpoints

### PR DESCRIPTION
**Pull request by jonasteuwen**
This is a draft for a pull request to download models from URLs. We need some way to also initialize a model easily. Right now loading from the checkpoint as an URL should work.

Needs to check
- [x] Calgary-Campinas masks need to download too, and so need to be uploaded?
- [x] Add a test for the downloads
- [x] Does initializing the model from the URL work properly now?

### CODE REVIEW

**Comment by jonasteuwen** [comment]
Failure of the tox test is slightly random. Can you review @georgeyiasemis @jonatanferm

File: tools/train_model.py [hunk 1]
```diff
@@ -258,13 +292,13 @@ def check_train_val(key, name):
...
-        type=pathlib.Path,
+        type=file_or_url,
     )
     parser.add_argument(
         "--initialization-checkpoint",
-        type=pathlib.Path,  # noqa
+        type=file_or_url,
         help="If this value is set to a proper checkpoint when training starts, "
         "the model will be initialized with the weights given. "
         "No other keys in the checkpoint will be loaded. "
```
Comment (georgeyiasemis) [inline tools/train_model.py:304]: I would add a bit more elaborate description about loading from directory or downloading from a url. (Maybe in the README?)

**Review by georgeyiasemis — APPROVED** [review]
Looks good in general. We maybe need to add some instructions on how to train a model or run inference with the new changes.

File: tools/train_model.py (hunk 1 above)
Comment (jonasteuwen) [reply tools/train_model.py:304]: I have adjusted it.

**Pull request merged**